CONTRIBUTING.md

```diff
@@ -53,24 +53,60 @@ For other ways to use `tldrl`, such as linting an entire directory, check out (w
 [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/master/pages/common/tldrl.md)
 
 ### Token syntax
-User-provided values should use the `{{token}}` syntax in order to allow clients
-to highlight them. 
+User-provided values should use the `{{token}}` syntax
+in order to allow clients to highlight them.
+Stick to [`snake_case`](https://en.wikipedia.org/wiki/Snake_case)
+when there are more than a word in the token.
```

Comment:
I agree it's more verbose than needed, but I like the explicitness. How about "use snake_case for multi-word tokens" instead?